addons/mail/static/src/js/services/mail_notification_manager.js

```diff
@@ -55,7 +55,8 @@ MailManager.include({
      */
     _handleChannelNotification: function (params) {
         if (params.data && params.data.info === 'typing_status') {
-            this._handleChannelTypingNotification(params.channelID, params.data);
+            // typing notification is no longer supported by mail manager
+            return;
```

Comment:
Just remove the condition entirely, the goal is to get rid of old code.

Reply:
we have to return, or reception of this notification on longpolling will fallback as a channelmessage notification, and it will crash